ota-plus-web/app/reactapp/src/components/devices/ContentPanel.jsx

```diff
@@ -88,14 +89,22 @@ class ContentPanel extends Component {
 
   render() {
     const { stores, changeFilter, showDeleteConfirmation, showEditName, addNewWizard } = this.props;
-    const { devicesStore, featuresStore, groupsStore } = stores;
+    const { featuresStore, devicesStore, groupsStore } = stores;
     const { alphaPlusEnabled } = featuresStore;
+    const { 
+      devices,
```

Comment:
Please try to keep alphabetical order as possible.

Reply:
Done